libraries/engage/locations/components/FulfillmentSheet/FulfillmentSheet.connector.js

```diff
@@ -19,12 +20,25 @@ function makeMapStateToProps() {
    * @param {Object} props The component props.
    * @returns {Object}
    */
-  return (state, props) => ({
-    locations: getProductLocations(state, props),
-    product: getProduct(state, props),
-    userInput: getUserFormInput(state),
-    fulfillmentPaths: getFulfillmentPaths(state),
-  });
+  return (state, props) => {
+    const {
+      firstName, lastName, mail: email, ...ignoreRest
```

Comment:
You do not need the `...ignoreRest` here, since you are destructuring. The "rest" will be left out automatically.